react/week2/todo/src/components/Todo.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react";
+
+function Todo(props) {
+  const [state, setDone] = useState([...props.items]);
```

Comment:
It is convention that the name of your setState method includes the name of your state. So eg. `const [puppy, setPuppy] = useState('wiener dog')`
In your case probably the most descriptive name for these items would be `todos`